packages/mantine/src/components/collection/__tests__/Collection.spec.tsx

```diff
@@ -38,7 +38,7 @@ describe('Collection', () => {
             });
             return (
                 <Collection<string> getItemId={getItemId} newItem="" {...form.getInputProps('fruits')}>
-                    {(name) => <span data-testid={'item'}>{name}</span>}
+                    {(name) => <span data-testid="item">{name}</span>}
```

Comment:
I think we don't those anymore since we now add the test id automatically

```suggestion
                    {(name) => <span>{name}</span>}
```